midi_app_controller/gui/binds_editor.py

```diff
@@ -364,85 +378,92 @@ def __init__(
         """
         super().__init__()
 
+        self.setWindowTitle("Edit Binds")
         self.binds = binds.copy(deep=True)
         self.save_binds = save_binds
 
+        name_label = QLabel("Name:")
```

Comment:
`name_label` is not used anywhere else so maybe just `name_layout.addWidget(QLabel("Name:"))`. It's done this way in the rest of the code.